shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java

```diff
@@ -690,14 +690,20 @@ protected boolean shouldAttachEngineToActivity() {
    * <p>The default implementation looks {@code <meta-data>} called {@link
    * FlutterActivityLaunchConfigs#HANDLE_DEEPLINKING_META_DATA_KEY} within the Android manifest
    * definition for this {@code FlutterFragmentActivity}.
+   *
+   * <p>Defaults to {@code true}.
    */
   @VisibleForTesting
   protected boolean shouldHandleDeeplinking() {
```

Comment:
Should this code go into a shared location since it is identical?

Reply:
I see there half of the functions in FlutterActivity.java and FlutterFragmentActivity.java are identical, so I was just following the pattern.  Maybe FlutterActivity and FlutterFragmentActivity need a refactor to move all identical code into a shared location?

Reply:
I still think this code should not be duplicated

Reply:
that's fair, i moved shared code into [FlutterActivityLaunchConfigs.java]